src/components/ContactsView.js

```diff
@@ -35,6 +35,9 @@ function ContactsView() {
         <a href={`https://twitter.com/${contact.twitter}`} target="_blank">
           {contact.twitter}
         </a>}
+        <Link to={`/contacts/${contact.id}/edit`} >
+          <p>Edit</p>
+        </Link>
```

Comment:
Like how you made an edit link in the contacts view page that redirects user to the edit page